src/modules/Dashboard/pages/Applet/Applet.tsx

```diff
@@ -14,13 +14,13 @@ export const Applet = () => {
   const location = useLocation();
 
   const appletTabs = useAppletTabs();
-  const { appletId } = useParams();
+  const { appletId, respondentId } = useParams();
 
   const { result: appletData } = applet.useAppletData() ?? {};
   const appletLoadingStatus = applet.useResponseStatus();
   const removeAppletData = useRemoveAppletData();
 
-  const hiddenHeader = location.pathname.includes('dataviz');
+  const hiddenHeader = !!respondentId || location.pathname.includes('dataviz');
```

Comment:
Could you please consider replacing the check with a more specific one? This is important because there could be other pages that also use ``respondentId`` in the ``location.pathname``.